src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1824,6 +1824,15 @@ properties:
                 type: string
                 fits_keyword: R_DRZPAR
                 blend_table: True
+          emicorr:
+            title: MIRI EMI correction reference file name
```

Comment:
minor nit, but for consistency with other entries, the title line here should be "MIRI EMI reference file information", since this object block is for the whole set of info about the file (even though we only have 1 lower-level attribute, i.e. the name of the file)